src/frontend/components/data-table/lib/component.js

```diff
@@ -290,21 +290,21 @@ class DataTableComponent extends Component {
     const $searchInput = $(`<input class='form-control form-control-sm' type='text' placeholder='Search' value='${searchValue}'/>`)
     $searchInput.appendTo($('.input', $searchElement))
     if (col.typeahead_use_id && searchValue) {
+      console.log("searchInput: ", $searchInput)
```

Comment:
Looks good, I am just wondering if this console log can be removed?

Reply:
Damn, thought I got them all! :wink: